pages/submission/[id].tsx

```diff
@@ -30,12 +47,20 @@ export default function SubmissionPage(props) {
       <div className="min-w-full mt-32">
         <div className="flex justify-center min-w-full mb-8">
           <div className="flex flex-col w-80">
-            <div className="w-full h-60 flex-none relative">
-              <Image
-                src={"/default_submission_image.jpeg"}
-                layout="fill"
-                alt="submission image"
-              />
+            <div className="w-full h-60 relative">
+              {submissionType === "video/quicktime" ?
+                <video
+                  className="absolute bottom-0"
+                  src={submission.mediaURI}
+                  ref={videoEl}
+                />
+                :
+                <Image
+                  src={"/default_submission_image.jpeg"}
+                  layout="fill"
+                  alt="submission image"
+                />
+              }
```

Comment:
Does doing this cause the default image to flash before it re-renders with the video?